### PR TITLE
Cuong

### PR DESCRIPTION
**Pull request by Cuong15061999**
Create model mongo

### CODE REVIEW

File: app/config/mongodb.py
Comment (pmhien2703) [inline app/config/mongodb.py:1]: move connection string into env

File: app/api/userRoutes.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from fastapi import APIRouter
+
+usersRouter = APIRouter()
```
Comment (pmhien2703) [inline app/api/userRoutes.py:3]: correct naming convention

**Comment by Cuong15061999** [comment]
@pmhien2703 please review again newest MR

File: app/utils/conversations_serialize.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+from utils.users_serialize import individual_user, list_users
+
+def individual_conversation(conversation) -> dict:
```
Comment (pmhien2703) [inline app/utils/conversations_serialize.py:3]: Define model for parameter @Cuong15061999

**Pull request merged**